backend/alembic/env.py

```
@@ -65,15 +65,25 @@ def run_migrations_online() -> None:
         poolclass=pool.NullPool,
+        url=url,
     )
 
     # Check if this is an async engine
     if isinstance(connectable, AsyncEngine):
-        asyncio.run(run_async_migrations(connectable))
+        # Check if there's already a running event loop
+        try:
+            asyncio.get_running_loop()
+            # If we get here, there's already a running loop, so schedule the coroutine
+            asyncio.create_task(run_async_migrations(connectable))
```

Ruff (0.13.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)